src/features/DrugChartSlider/utils/DrugChartSliderUtils.js

```diff
@@ -102,3 +114,61 @@ export const getUTCTimeEpoch = (time, enable24HourTimers, scheduledDate) => {
   const utcTimeEpoch = moment.utc(localTime).unix();
   return utcTimeEpoch;
 };
+
+export const epochTo24HourTimeFormat = (epochSeconds) => {
+  const date = new Date(epochSeconds * 1000);
+  const hours = date.getHours().toString().padStart(2, "0");
+  const minutes = date.getMinutes().toString().padStart(2, "0");
+  const formattedTime = `${hours}:${minutes}`;
+  return formattedTime;
+};
+
+export const epochTo12HourTimeFormat = (epochSeconds) => {
+  const date = new Date(epochSeconds * 1000);
+  const hours = date.getHours();
+  const minutes = date.getMinutes().toString().padStart(2, "0");
+  const amOrPm = hours >= 12 ? "PM" : "AM";
+  const formattedHours = (hours % 12 || 12).toString().padStart(2, "0");
+  const formattedTime = `${formattedHours}:${minutes} ${amOrPm}`;
```

Comment:
for formatting the dates use moment js methods, moments js has methods to convert epoch to desired date format, no need of writing these logics on our end